.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*	@pdcastro @thgreasi @CameronDiver @hedss
+*	@CameronDiver @hedss @pdcastro @srlowe @thgreasi
```

Comment:
Added @srlowe to the CODEOWNERS file, so he's automatically included as a reviewer in CLI pull requests. The list was reordered alphabetically.